src/docs/development/data-and-backend/state-mgmt/simple.md

```diff
@@ -0,0 +1,327 @@
+---
+title: Simple app state management
+prev:
+  title: Ephemeral versus app state
+  path: /docs/development/data-and-backend/state-mgmt/ephemeral-vs-app
+next:
+  title: List of approaches
+  path: /docs/development/data-and-backend/state-mgmt/options
+---
+
+Now that you know about [declarative UI programming](/docs/development/data-and-backend/state-mgmt/declarative) and the difference between [ephemeral and app state](/docs/development/data-and-backend/state-mgmt/ephemeral-vs-app), you are ready to learn about simple app state management.
+
+This page describes an approach particularly appropriate to those new to Flutter, without a pre-existing preference for a particular state management approach. Developers with strong background in state management from other reactive frameworks, such as Redux or Rx, will find packages and tutorials listed on the [following page](/docs/development/data-and-backend/state-mgmt/options).
```

Comment:
If your intent with doc is (at least in part) to give people a "default" approach to state management via ScopedModel, consider being more explicit about it right up front:

* You're about to see us use a package called ScopedModel
* It uses Inherited Widgets and a Model object
* If you don't have a strong reason to choose Redux/Rx/BLoC, this is probably the approach you should start with.
  - Because it's easy to understand, but flexible and powerful
  - And it doesn't use much code

You've already got part of that in place (and cover the rest later, I think), but a little more detail right up front might help ground the reader before they progress.

Reply:
ack